TestAssets/TestProjects/SlnFileWithSolutinItemsInNestedFolders/ConsoleApp2/ConsoleApp2.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we can skip adding this file.  It's not needed by the test (`dotnet sln remove` can remove projects from a solution without the project being on disk).